include/gz/sim/Util.hh

```diff
@@ -267,18 +267,18 @@ namespace gz
         Entity _entity, const EntityComponentManager &_ecm);
 
     /// \brief Environment variable holding resource paths.
-    const std::string kResourcePathEnv{"IGN_GAZEBO_RESOURCE_PATH"};
+    const std::string kResourcePathEnv{"GZ_SIM_RESOURCE_PATH"};
```

Comment:
Is this variable being tick-tocked? We should still accept `IGN_GAZEBO_RESOURCE_PATH` with warnings.

Reply:
Warning will emit via https://github.com/gazebosim/gz-common/pull/365/commits/21c47bb50f5d3397bfc1c00f647b07ac547d4fc2

https://github.com/gazebosim/gz-sim/pull/1518/commits/ac92c2ca29b222d881e5819611aec99eb26e071f

Reply:
Smart!